files/build_templates/sonic_debian_extension.j2

```diff
@@ -724,6 +724,13 @@ sudo LANG=C chroot $FILESYSTEM_ROOT sonic-package-manager install {{ name }}=={{
 sudo LANG=C chroot $FILESYSTEM_ROOT sonic-package-manager install --from-tarball {{ path }} {{ get_install_options(set_owner, enabled) }}
 {% endfor -%}
 
+{% if install_third_party_debs -%}
+TEMP_DIR="$(sudo LANG=C chroot $FILESYSTEM_ROOT mktemp -d)"
+sudo wget {{ install_third_party_debs.strip().replace(",", " ") }} --directory-prefix="$FILESYSTEM_ROOT/$TEMP_DIR"
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT bash -c "apt-get install -y \"$TEMP_DIR\"/*"
```

Comment:
@stepanblyschak do we really need `\"$TEMP_DIR\"` ? AFAIK, `mktemp` by default produces non-spaces path.

Reply:
Haven't seen this is mentioned in man page

Reply:
@stepanblyschak please have a look at this:
1. mktemp:
https://refspecs.linuxfoundation.org/LSB_5.0.0/LSB-Core-generic/LSB-Core-generic/mktemp.html
http://manpages.ubuntu.com/manpages//precise/en/man3/mktemp.3posix.html
https://www.ibm.com/docs/en/zos/2.3.0?topic=functions-mktemp-make-unique-file-name
2. Portable Filename Character Set:
https://pubs.opengroup.org/onlinepubs/9699919799.2008edition/basedefs/V1_chap03.html#tag_03_276
https://www.ibm.com/docs/en/zos/2.3.0?topic=locales-posix-portable-file-name-character-set
https://dwheeler.com/essays/fixing-unix-linux-filenames.html

Reply:
Done